configs/babel.js

```diff
@@ -86,7 +86,7 @@ module.exports = function (api) {
 			// '@babel/plugin-proposal-json-strings'
 
 			require('babel-plugin-dev-expression'),
-			env === 'test' && !es5Standalone && require('babel-plugin-dynamic-import-node').default,
+			env === 'test' && !es5Standalone && require('babel-plugin-dynamic-import-node').default, // eslint-disable-line
```

Comment:
Note: this line was added to fix travis fail
https://app.travis-ci.com/github/enactjs/storybook-utils/builds/257147611

Reply:
At the beginning of the PR, it was normally travis pass.
https://app.travis-ci.com/github/enactjs/storybook-utils/builds/256846826

What is the cause? Can't you fix it?

Reply:
I tried updating npm-shrinkwrap.json but the travis fail persists.
I don't know what is causing it since I only reverted the package versions. It seems to pass only if we have `//eslint-disable-line`

Reply:
That's because `babel-plugin-dynamic-import-node` was removed.
Please install the new one.
(npm install babel-plugin-dynamic-import-node)

Reply:
Okay